Storage/ThirdParty/SwiftData.swift

```diff
@@ -840,13 +831,11 @@ open class ConcreteSQLiteDBConnection: SQLiteDBConnection {
                 
                 log.debug("Finished moving database \(baseFilename) successfully.")
             } catch let error as NSError {
-                log.error("Unable to move \(baseFilename) to another location. \(error)")
-                SentryIntegration.shared.sendWithStacktrace(message: "Unable to move \(baseFilename) to another location. \(error)", tag: "SwiftData", severity: .error)
+                Sentry.shared.sendWithStacktrace(message: "Unable to move db to another location", tag: SentryTag.swiftData, severity: .error, description: "\(baseFilename) to another location. \(error)")
             }
         } else {
             // No backup was attempted since the database file did not exist.
-            log.error("The database file \(baseFilename) has been deleted while previously in use.")
-            SentryIntegration.shared.sendWithStacktrace(message: "The database file \(baseFilename) has been deleted while previously in use.", tag: "SwiftData", severity: .info)
+            Sentry.shared.sendWithStacktrace(message: "The database file has been deleted while previously in use.", tag: SentryTag.swiftData, description: "filename \(baseFilename)")
```

Comment:
Change to `description: "DB file '\(baseFilename)'. \(error.localizedDescription)"`